### ISSUE TITLE
Create up-to-date docs about Bisq 2

### ISSUE THREAD

**Issue by HenrikJannsen** [bisq-network/bisq2#950]
There is no up to date documentation about Bisq 2. 
The [readme](https://github.com/bisq-network/bisq2/blob/main/README.md) is outdated as well as the linked pages from there as well as the various discussion or proposal pages.

Here a list of documents and scopes which would be good to cover:
- High level overview about goals and motivation (similar and updated to the links in the readme file)
- High level overview about the concepts from user point of view (options and features) and from architecture (modules, layers)
- Overview about the core modules and layers: P2P network, multiple mix network support, multiple wallet support, identity concept, reputation system, protocol execution framework, oracle nodes,...
- Overview about concepts how to implement a mobile app for Bisq Easy (Android vs. iOS) 
- Overview about the infrastructure of nodes and how it connects to Bisq 1 and the DAO (bridge nodes), APIs, Webinterface for selfhosted node projects, Mobile app
- Roadmap for protocols we have in mind (new ideas about LN-> L-BTC -> LN is not documented at all yet)
- Dev docs, how to setup, run Bisq 2, code styles and concepts 

Where it makes sense a graphical overview would be good as well. As most parts are quite complex, I think to split it in a high level overview to get a quick idea and a detail spec would be good.

**Comment by counterweightoperator** [comment]
I agree that some of these are very complex, and I think also quite different in their goals and audiences. It probably makes sense to split this.

A few questions on some of the listed points:
> Overview about concepts how to implement a mobile app for Bisq Easy (Android vs. iOS)

I'm not sure I follow this one. If I understand correctly, you mean some kind of architecture summary aimed for Android/iOS devs that would like to understand how to integrate their app with Bisq?

> Roadmap for protocols we have in mind (new ideas about LN-> L-BTC -> LN is not documented at all yet)

What's your idea here? A high level development plan? Individual docs for each different trade protocol?

**Comment by HenrikJannsen** [comment]
Hi @counterweightoperator 

> I'm not sure I follow this one. If I understand correctly, you mean some kind of architecture summary aimed for Android/iOS devs that would like to understand how to integrate their app with Bisq?

I meant an overview / architecture summary about how to implement Bisq Easy on Android.


> What's your idea here? A high level development plan? Individual docs for each different trade protocol?

I just added recently the LN-Fiat protocol to the trade apps:
<img width="1522" alt="Screenshot 2023-07-10 at 01 48 53" src="https://github.com/bisq-network/bisq2/assets/116298498/e8d2da4e-c24d-4f99-8321-0def64c1415c">

A roadmap could be along those lines...
